troupon/static/scss/modules/_utils.scss

```diff
@@ -0,0 +1,77 @@
+
+// ==========================================================================
+//  Special Utility Styles
+// ==========================================================================
+
+
+@mixin fluid-width {
+    width: 100%;
+}
+@mixin fluid-width-xs {
+    @media (max-width: 768px) {
+        @include fluid-width;
+    }
+}
+
+@mixin v-middle {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%); 
+}
+@mixin contents-v-middle ($min-height:55px) {
+    position: relative;
+    display: inline-block;
+    min-height: $min-height;
+
+    & > * {
+        @include v-middle;
+    }
+}
+
+
+@mixin shaded {
+	background-color: color(shaded-bg);
+}
+@mixin shaded-border {
+	@include shaded;
+
+	border-top: 1px solid color(shaded-border);
+    border-bottom: 1px solid color(shaded-border);
+}
+
+
+@mixin divider {
+    width: 100%;
+    height: 1px;
+    margin: 0px 0px;
+    overflow: hidden;
+    background-color: color(divider);
+}
+@mixin divider-v {
+    @include divider;
+    width: 1px;
+    margin: 0px;
+    vertical-align: middle;
+}
+
+@mixin badge {
+    padding: 5px 8px;
+    color: color(badge-text);
+    background-color: color(badge-bg);
+}
+
+
+
+@mixin unstyled-list-mixin {
+    list-style: none outside none;
+    padding: 0px 0px;

```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `0px` instead of `0px 0px`<br>`0px` should be written without units as `0`